qemu/tests/usb_storage.py

```diff
@@ -134,6 +148,7 @@ def _check_serial_option(serial, regex_str, expect_str):
         if serial not in ["EMPTY_STRING", "NO_EQUAL_STRING"]:
             # Verify in guest when serial is set to empty/null is meaningless.
             _verify_string(serial, output, [serial])
+        _umount_usb(session)
```

Comment:
Hello @balamuruhans . I thought about it, having an if clause to check whether unmount usb or not.
But since after usb disk was made file system, it will be mounted after reboot. if one set `unmount_usb_check = no `,the case will run into errors.

So, it has to do `_umount_usb(session)`. I use a `chk_umount_cmd` in that function, if there is a  mounted usb disk , do umount. if not, it will perform subsequent steps.